kdestandardize.ado

```diff
@@ -1399,18 +1412,51 @@ prog def kdestandardize
 									
 	} // End of handling of the Equity_Measure  variable
 	
-	// Handles instances of the  variable
-	if `: list posof "" in x' != 0 {
-	
-	} // End of handling of the  variable
-	
-	// Handles instances of the  variable
-	if `: list posof "" in x' != 0 {
-	
-	} // End of handling of the  variable
-	
-	
-	  
+	// Handles instances of the EQ_LABEL variable
+	if `: list posof "EQ_LABEL" in x' != 0 {
+		qui: replace EQ_LABEL=cond(`rx'(EQ_LABEL, "Exemplary/ Accomplished", 1), "1", ///
+							  cond(EQ_LABEL=="High/Expect","2",              ///
+							  cond(EQ_LABEL=="School-Level","3",             ///
+							  cond(EQ_LABEL=="Strongly Agree/Agree","4"      ///
+							  cond(EQ_LABEL==" ","5")))))                    ///
+		qui: rename EQ_LABEL eqlabel
+		qui: destring eqlabel,replace ignore("*,-R %")
+		la val eqlabel eqlabel
+		la var eqlabel Equity Label
+	} // End of handling of the EQ_LABEL variable
+	
+	// Handles instances of the EQ_PCT variable
+	if `: list posof "EQ_PCT" in x' != 0 {
+	qui: rename EQ_PCT eqpct
```

Comment:
Use all lowercase variable names `eq_pct` instead of `EQ_PCT`.  The `kdecombo` command imports all of the variable names in lower case so these tests wouldn't find the variable intended.